python/hail/dataset.py

```diff
@@ -2003,24 +2003,36 @@ def hardcalls(self):
         pargs = ['hardcalls']
         return self.hc._run_command(self, pargs)
 
-    def ibd(self, output, maf=None, unbounded=False, min=None, max=None):
+    def ibd(self, maf=None, unbounded=False, min=None, max=None):
         """Compute matrix of identity-by-descent estimations.
 
         **Examples**
 
-        To estimate and write the full IBD matrix to *ibd.tsv*, estimated using minor allele frequencies computed from the dataset itself:
+        To calculate a full IBD matrix, using minor allele frequencies computed
+        from the dataset itself:
 
-        >>> vds.ibd('output/ibd.tsv')
+        >>> vds.ibd()
 
-        To estimate IBD using minor allele frequencies stored in ``va.panel_maf`` and write to *ibd.tsv* only those sample pairs with ``pi_hat`` between 0.2 and 0.9 inclusive:
+        To calculate an IBD matrix containing only pairs of samples with
+        ``pi_hat`` in [0.2, 0.9], using minor allele frequencies stored in
+        ``va.panel_maf``:
 
-        >>> vds.ibd('output/ibd.tsv', maf='va.panel_maf', min=0.2, max=0.9)
+        >>> vds.ibd(maf='va.panel_maf', min=0.2, max=0.9)
 
         **Details**
 
-        The implementation is based on the IBD algorithm described in the `PLINK paper <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838>`_.
+        The implementation is based on the IBD algorithm described in the `PLINK
+        paper <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838>`_.
 
-        :py:meth:`~hail.VariantDataset.ibd` requires the dataset to be bi-allelic (otherwise run :py:meth:`~hail.VariantDataset.split_multi`) and does not perform LD pruning. Linkage disequilibrium may bias the result so consider filtering variants first.
+        :py:meth:`~hail.VariantDataset.ibd` requires the dataset to be
+        bi-allelic (otherwise run :py:meth:`~hail.VariantDataset.split_multi`)
+        and does not perform LD pruning. Linkage disequilibrium may bias the
+        result so consider filtering variants first.
+
+        The resulting :py:class:`.KeyTable` keys are pairs of samples with type
```

Comment:
Put the KeyTable signature in a code block.  Keys + values is the wrong abstraction, give the full key table schema and then the (ordered) list of keys.